docs/specification.md

```diff
@@ -681,6 +677,27 @@ Parameter (in Michelson):
 - Fails with `NOT_ENOUGH_FROZEN_TOKENS` if the author does not have enough tokens
   that can be burned.
 
+
+### **unstake_vote**
+
+```ocaml
+type proposal_key = bytes
+type unstake_vote_param = [proposal_key]
+
+Unstake_vote of unstake_vote_param
+```
+
+Parameter (in Michelson):
+```
+(list %unstake_vote bytes)
+```
+
+- Unfreeze voter's tokens for proposals that are already flushed.
```

Comment:
>Unfreeze voter's tokens

Probably we should also say "Unstake" here as well..

Reply:
Nice catch, updated this.